lib/Converter.js

```diff
@@ -45,7 +45,7 @@ class Converter
     {
         return puppeteer.launch({
             ignoreHTTPSErrors: true,
-            args: ['--no-sandbox', '--disable-web-security']
+            args: ['--no-sandbox', '--disable-web-security', '--font-render-hinting=none']
```

Comment:
What issue is this intended to fix?

Reply:
Unfortunately I can't share any screenshots, but it fixes this issue: https://github.com/puppeteer/puppeteer/issues/2410